src/transport_performance/gtfs/validation.py

```diff
@@ -838,8 +853,12 @@ def _plot_summary(
         _check_column_in_df(df=summary_df, column_name=target_column)
         _check_column_in_df(df=summary_df, column_name=day_column)
 
-        # convert column type for better graph plotting
+        # convert column type for better graph plotting, use desc
+        ROUTE_LKP = get_saved_route_type_lookup()
         summary_df["route_type"] = summary_df["route_type"].astype("object")
+        summary_df["route_type"] = summary_df["route_type"].apply(
+            lambda x: _get_route_type_desc(ROUTE_LKP, x)
+        )
```

Comment:
By refactoring to:
```
summary_df["route_type"] = summary_df["route_type"].astype(str)
summary_df = summary_df.merge(ROUTE_LKP, how="left", on="route_type")
```

we can save lines by deleting `def _get_route_type_desc(lkp, key):`

and the associated test within test_validation.py

Reply:
@r-leyshon This should be resolved now with commit 14c2f26